vendor.conf

```diff
@@ -89,6 +89,7 @@ sigs.k8s.io/yaml                                    9fc95527decd95bb9d28cc2eab08
 github.com/containerd/go-cni                        0d360c50b10b350b6bb23863fd4dfb1c232b01c9
 github.com/containernetworking/cni                  4cfb7b568922a3c79a23e438dc52fe537fc9687e # v0.7.1
 github.com/containernetworking/plugins              9f96827c7cabb03f21d86326000c00f61e181f6a # v0.7.6
+github.com/fsnotify/fsnotify                        4bf2d1fec78374803a39307bfb8d340688f4f28e
```

Comment:
can we use a versioned fsnotify? I see it's not had a new version tag since 18.. hmm Trying to get to a good place for go mod

Reply:
Yeah. The current master `4bf2d1fec78374803a39307bfb8d340688f4f28e` contains the pr https://github.com/fsnotify/fsnotify/pull/219, which used to prevent fd leak from cni call or other exec commands. It is good patch, but the latest tag `v1.4.7` doesn't include this one. 

So I still use `4bf2d1fec78374803a39307bfb8d340688f4f28e` commit here. Is it good choice?

Reply:
ping @cpuguy83

Reply:
Yes this is a good commit. The project does not have very much changes.
Drafting a new release for fsnotify now, just waiting on some other maintainers to OK it.

Reply:
And 4bf2d1fec78374803a39307bfb8d340688f4f28e will be the tagged commit.

Reply:
https://github.com/fsnotify/fsnotify/releases/tag/v1.4.8

Reply:
@cpuguy83 Thank you! will update it

Reply:
updated. but still no idea about node e2e testing